samples/server/petstore/php-symfony/SymfonyBundle-php/.openapi-generator/FILES

```diff
@@ -1,26 +1,26 @@
-./Api/ApiServer.php
-./Api/PetApiInterface.php
-./Api/StoreApiInterface.php
-./Api/UserApiInterface.php
-./Controller/Controller.php
-./Controller/PetController.php
-./Controller/StoreController.php
-./Controller/UserController.php
-./Model/ApiResponse.php
-./Model/Category.php
-./Model/Order.php
-./Model/Pet.php
-./Model/Tag.php
-./Model/User.php
-./Service/JmsSerializer.php
-./Service/SerializerInterface.php
-./Service/StrictJsonDeserializationVisitor.php
-./Service/SymfonyValidator.php
-./Service/TypeMismatchException.php
-./Service/ValidatorInterface.php
-./Tests/AppKernel.php
```

Comment:
please revert the changes in the `/FILES` files, as these cause the CI to fail.
unfortunately, I haven't investigated why this occurs.
cc @wing328

Reply:
Reverted and also updated to latest `master`